llvm/test/CodeGen/WebAssembly/interleave.ll

```diff
@@ -0,0 +1,361 @@
+; RUN: opt -mattr=+simd128 -passes=loop-vectorize %s | llc -mtriple=wasm32 -mattr=+simd128 -verify-machineinstrs -o - | FileCheck %s
```

Comment:
Is this really the right level of test if you are just changing TTI? This would go through both cost and scheduling models, for example. Maybe analysis test would be better?

Reply:
> just changing TTI?

When the change means the vectorizer acts in a completely different way, I think it's best that we have a test to monitor how the vectorizer continues to treat this input. It could change, but that is the point.

Reply:
This test will flag vectorizer changes that happen to produce a different output - I don't think the [tests](https://llvm.org/docs/DeveloperPolicy.html#test-cases) are intended to monitor changes in other components like that, maybe llvm-testsuite is a better place? Same here: https://github.com/llvm/llvm-project/pull/122896#discussion_r1957868703